components/tools/OmeroWeb/omeroweb/settings.py

```diff
@@ -1140,6 +1140,11 @@ def report_settings(module):
 # FEEDBACK_APP: 6 = OMERO.web
 FEEDBACK_APP = 6
 
+# For any given release of api, we may support
+# one or more versions of the api.
+# E.g. /api/v1.0/
+API_VERSIONS = ['0.1']
```

Comment:
just discussed in the office, I had a different understanding of what this var is doing. If that is a list of supported versions, I don't see the version anywhere to match and consume that list. Allowing multiple version numbers in urls in the same definition of url is kind of strange as how we are going to write code for both implementations? I would prefer to avoid logic like:

```
def view(request,...):
    if (version > (0,2)):
        do foo
    else:
        do bar
```

Pretty much, it looks to me that `api_version` could be an app. Something to discuss